### ISSUE TITLE
Python 3.8 not supported

### ISSUE THREAD

**Issue by matthew-d-jones** [mdavidsaver/p4p#49]
I get the following error on Ubuntu 20.04 when trying to use Python 3.8:
```
ImportError: venv/lib/python3.8/site-packages/p4p/_p4p.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN5epics8pvAccess20ConfigurationBuilder4_addERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES9_
```

P4P works as expected with Python 3.7 on the same system.

**Comment by mdavidsaver** [comment]
This is likely not py3.8 directly.  Rather I think the issue is that there is a 3.8 wheel for the `epicscorelibs` dependency, but not for `p4p`.  The wonders of c++ ABI compatibility (or lack thereof) means that you either need to do a local build of both or neither.

For the moment you should be able to make P4P work with py3.8 by doing following the [build from source recipe](https://mdavidsaver.github.io/p4p/building.html#build-as-python-package):

```
python -m pip install nose numpy
python -m pip uninstall epicscorelibs     # assuming it is already installed
python -m pip install --no-binary :all: p4p
```

I'll keep this issue open until 3.8 wheels are published, which should be soon.

**Comment by matthew-d-jones** [comment]
Thank you, that works.

**Comment by mdavidsaver** [comment]
If you have time, can you test [p4p 3.5.0a2](https://pypi.org/project/p4p/3.5.0a2/)?  This is a pre-release so you'll need to add '--pre'.

```
python -m pip install --pre p4p
```

**Comment by matthew-d-jones** [comment]
@mdavidsaver 
Tested with a fresh virtualenv (Python 3.8.2, Ubuntu 20.04):
```
Successfully installed epicscorelibs-7.0.3.99.4.0a3 nose-1.3.7 numpy-1.18.4 p4p-3.5.0a2 ply-3.11
```
and integration tests of my application pass, so looks good to me :+1:

**Comment by mdavidsaver** [comment]
Released [3.5.0](https://pypi.org/project/p4p/3.5.0/) including py3.8 wheels for linux/osx/windows.
